src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/StringToListConverter.shared.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Xamarin.CommunityToolkit.Extensions.Internals;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	/// <summary>
+	/// Returns a string array that contains the substrings in this string that are delimited by <see cref="Separator"/>.
+	/// </summary>
+	public class StringToListConverter : ValueConverterExtension, IValueConverter
+	{
+		/// <summary>
+		/// The string that delimits the substrings in this string.
+		/// </summary>
+		public string Separator { get; set; } = string.Empty;
+
+		/// <summary>
+		/// The strings that delimits the substrings in this string.
+		/// </summary>
+		public IList<string> Separators { get; } = new List<string>();
```

Comment:
I guess we can use IEnumerable<string> here

Reply:
The reason for using `IList` is because of the method Add() that is used in the approach, which does not exist in `IEnumerable`.